test/sql/copy/parquet/parquet_stats.test

```diff
@@ -206,3 +206,14 @@ query II
 select stats_min_value is null c0, stats_max_value is null c1 from parquet_metadata('__TEST_DIR__/test.parquet');
 ----
 false	false
+
+query II
```

Comment:
This is the first time I write test here, I don't know if this is ok. Would there any doc or guild for test?

Reply:
- sql test overview: https://duckdb.org/docs/stable/dev/sqllogictest/overview.html
- general testing: https://github.com/duckdb/duckdb/blob/main/CONTRIBUTING.md#testing